src/app/template-library/template-library.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="ion-page bg-white">
+  <ion-split-pane class="template-library-container" contentId="main">
+    <ion-menu contentId="main" class="menu border-0">
+      <div class="search-home-container">
+        <ion-searchbar color="light" class="template-search-bar"></ion-searchbar>
+      </div>
+      <h6 class="menu-header">Categories</h6>
+      <ion-content>
+        <ng-container *ngFor="let category of categories">
+          <p class="cursor-pointer" (click)="viewCategory(category)">{{ category.name }}</p>
+        </ng-container>
+      </ion-content>
+    </ion-menu>
+
+    <ion-content id="main">
+
+      <ion-fab class="ion-hide-lg-up" vertical="top" horizontal="start" slot="fixed">
+        <ion-fab-button color="light">
+          <ion-menu-button> </ion-menu-button>
+        </ion-fab-button>
+      </ion-fab>
+
+      <!--      <ion-menu-button> </ion-menu-button>-->
+      <ion-grid class="templates-grid-container">
+
+        <ion-row>
+          <ng-container *ngFor="let category of categories.slice(0,3)">
+            <ion-col>
+              <app-template-category-card [category]="category" (click)="viewCategory(category)"></app-template-category-card>
+            </ion-col>
+          </ng-container>
+        </ion-row>
+
+        <ion-row>
+          <ng-container *ngFor="let category of categories.slice(3,7)">
+            <ion-col>
+              <app-template-category-card [category]="category" (click)="viewCategory(category)"></app-template-category-card>
+            </ion-col>
+          </ng-container>
+        </ion-row>
+
+        <ng-container *ngIf="!loadingTemplates">
+          <ng-container *ngFor="let categorisedTemplate of categorisedTemplates">
+            <ng-container *ngIf="categorisedTemplate.templates.length > 0">
+
+              <ion-row>
+                <ion-col>
+                  <h2 class="category-heading cursor-pointer" (click)="viewCategory(categorisedTemplate.category)">{{categorisedTemplate.category.name}} <ion-icon class="align-middle" name="arrow-forward-outline"></ion-icon></h2>
```

Comment:
![image](https://user-images.githubusercontent.com/14796879/113248563-caab6000-92ef-11eb-9ebe-97695bc12b5e.png)
The category header seems to be closer to the top elements instead of the templates. Will need to modify those spaces a bit.